src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_acl_permission.erl

```diff
@@ -1,7 +1,8 @@
 %% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92-*-
 %% ex: ts=4 sw=4 et
 %% @author Douglas Triggs <doug@chef.io>
-%% Copyright 2014 Chef Software, Inc. All Rights Reserved.
+%% @auther Mark Mzyk <mm@chef.io)
+%% Copyright 2014-2015 Chef Software, Inc. All Rights Reserved.
```

Comment:
Outside of changing names, this was just a copy paste and I didn't think about the details, so I'll update. Thanks for catching this.